services/web/server/src/simcore_service_webserver/resource_manager/websocket_manager.py

```diff
@@ -90,4 +90,10 @@ def _resource_key(self) -> Dict[str,str]:
 @contextmanager
 def managed_resource(user_id: str, client_session_id: str, app: web.Application) -> WebsocketRegistry:
     registry = WebsocketRegistry(user_id, client_session_id, app)
-    yield registry
+    try:
+        yield registry
+    except Exception:
+        log.exception("Error in web-socket for user:%s, session:%s", user_id, client_session_id)
+        raise
```

Comment:
the error message is not correct. the websocket registry is the registry mapping websockets and client_session_id. if an error happen here it does not mean the websocket has an issue, but more something to do with redis, or something with the mapping

Reply:
Fine. I understand. It was some placeholder ... in any case, the question here is whether we can handle some exceptions in the context manager ...